tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py36,py37,py38,py39,py310
+envlist = py38,py39,py310,py311
 [testenv]
 deps = -rrequirements.txt
 commands=
     python ./scripts/license_verify.py
     flake8 --statistics --append-config=.flake8 knack
-    pylint knack --rcfile=.pylintrc -r n -d I0013
+    pylint knack --rcfile=.pylintrc --reports n --disable I0013
```

Comment:
Use arguments' full name for clarity.

It's very likely we need to remove `--disable I0013` in the near future, as pylint 3.0.0b1 disables it by default:

> This message is disabled by default. To enable it, add file-ignored to the enable option.
> -- https://pylint.pycqa.org/en/latest/user_guide/messages/information/file-ignored.html